src/App.vue

```diff
@@ -354,6 +354,15 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
     )
   }
 
+  /** True if there is filing data for corrections - corrections has a single filing schedule. */
+  get correctionHasFilingData () : boolean {
+    return Boolean(
+      this.getFilingData?.length > 0 &&
+      this.getFilingData[0].filingTypeCode &&
+      this.getFilingData[0].entityType
+    )
+  }
+
```

Comment:
Quoting from your last comment "... because the base filing has an additional fee schedule for a NAME CHANGE...". By "additional fee schedule", do you mean here?

Reply:
Corrections only has a single filing schedule 

![image](https://user-images.githubusercontent.com/3484109/183777294-cc0a011d-f41e-457f-b1a4-e2d46fbc1db3.png)

Has two different fee schedules (depending on corp_type):

![image](https://user-images.githubusercontent.com/3484109/183777337-4cb8721a-52f8-4e0f-9722-2427c9c930b8.png)

Reply:
Okay. I understand now. Thanks.